rules/Proj0001.md

```diff
@@ -0,0 +1,7 @@
+# Proj0001: .NET project file could not be located
+Roslyn officially does not support the analysis on .NET project files. Therefor
+all rules in this package try to resolve the project file on disk based on
```

Comment:
This sentence reads a bit weird with the two times 'on'.

Reply:
I dropped "on disk"